utils/getInvoices/index.ts

```diff
@@ -212,7 +212,7 @@ export const getInflicionInvoice = ({
     const prevPlacing = prevInvoicesCollection[ServiceType.Placing]
```

Comment:
Let's update this logic to load inflicion even if no `prevService` was provided.

```
if (company?.inflicion) {
  if (isEmpty(prevService?.inflicionPrice) /* maybe with other conditions, idk*/) {
    return { Inflicion = 0 }
  }
  
  return { Inflicion = formula }
}
```

Reply:
Update tests related to this case